src/vulnpy/trigger/nosqli.py

```diff
@@ -0,0 +1,50 @@
+from pymongo import MongoClient
+import os
+
+
+def mongo_db_exists():
+    client = MongoClient(serverSelectionTimeoutMS=20)
+    db = client.pymongo_test
+
+    try:
+        db.posts.count_documents({})
+    except Exception:  # pragma: no cover
+        return False
+    return True
+
+
+def create_mongo_client(**kwargs):
+    mongo_host = [os.environ.get("MONGODB_HOST", "localhost:27017")]
+    if mongo_db_exists():
+        return MongoClient(host=mongo_host, **kwargs)
+    else:
+        print("Pymongo service is not running.")  # pragma: no cover
+
+
+client = create_mongo_client()
+db = client.pymongo_test_db
```

Comment:
😛  this causes an error when `create_mongo_client` returns None

You can test this the same way I'm testing it by running `make <framework>` from the repo root. If you shut down mongo locally and run for example `make flask` you'll be able to see if the app starts or not

Reply:
Thanks for spotting this, nice one.